dogs.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>ADOPTADOG</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  </head>
+  <body>
+    <div class="header">
+      <h1>ADOPTADOG</h1>
+      <h2>Seattle's premiere dog adoption agency</h2>
+    </div>
+
+    <div class="navbar">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+          <li><a href="checkout.html">Checkout</a></li>
+        </ul>
+    </div>
+
+    <div class="allthedogs">
+      <h1>Here are all our Dogs</h1>
+      <img src="images/murphy-card.jpg" alt="a dog name murphy" width="33%">
```

Comment:
Looks like you're missing the dog information. Also, it's fine for now, but you'll want to handle all styling in css files.